src/sentry/incidents/subscription_processor.py

```diff
@@ -335,51 +375,51 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
 
         if aggregation_value is not None:
             if has_metric_alert_processing:
-                packet = QuerySubscriptionUpdate(
-                    entity=subscription_update.get("entity", ""),
-                    subscription_id=subscription_update["subscription_id"],
-                    values={"value": aggregation_value},
-                    timestamp=self.last_update,
-                )
+                if self.alert_rule.detection_type == AlertRuleDetectionType.DYNAMIC:
+                    packet = QuerySubscriptionUpdate(
```

Comment:
🤔 should we create a second data packet type definition for this? Then we can do something like: `MetricIssueDataPacket = QuerySubscriptionUpdate | AnomalyDetectionUpdate`